.changeset/wide-ghosts-own.md

```diff
@@ -0,0 +1,12 @@
+---
+"@pnpm/core": minor
+"@pnpm/config": minor
+"@pnpm/plugin-commands-installation": patch
+"@pnpm/plugin-commands-patching": patch
+---
+
+Add new setting `catalogMode` with options `strict`, `prefer`, and `manual` for controlling if and how dependencies are added to the default catalog.
```

Comment:
nitpick: @zkochan might have a different opinion, but I usually like to write this without the imperative mood since I find it strange in release notes.

```suggestion
A new `catalogMode` setting is available for controlling if and how dependencies are added to the default catalog. It can be configured to several modes:
```

The current imperative mood (e.g. "_Add new setting..._") makes sense for git commits, but I personally don't use imperative mood for release notes.

Reply:
I don't have a different opinion.